style.css

```diff
@@ -115,9 +107,113 @@ a:active {
   transform: scale(0.85);
 }
 
+/* FOOTER */
+footer {
+  display: flex;
+  flex-direction: column;
+  background: #6070ff url(images/contact\ form\ shapes-mobile@2x.svg);
+  background-size: 60%;
+  background-position: top 18px right 0;
+  background-repeat: no-repeat;
+  border-top-left-radius: 80px;
+  padding: 24px 24px 0;
+  color: var(--neutrals-n-0, #fff);
+  align-items: center;
+}
+
+footer > * {
+  max-width: 327px;
+}
```

Comment:
- You did well making the contact form responsive, but as the previous reviewer said:
 ` All elements in the contact form section currently have fixed width and height. Instead of using fixed values, it's recommended to use responsive values in your CSS rules, such as percentages (e.g., width: 80%), to ensure that the layout adjusts properly on different devices and screen sizes.`. You have a fixed width for all child components of the footer section. Please use a responsive measurement unit e.g `width: 100vw`. Please fix this 👍 👍

Reply:
HI @mckent05 . Thanks for reviewing my code.
To be honest, I struggled to find a way to satisfy your request, but I don't think there's any way to do what you're asking. 
`max-width` is very different than `width`. It's a guideline that's meant to be responsive, as opposed to `width`, which is fixed. In a way, `max-width` is a kind of media query, and media queries don't use percentages. Using percentages would simply not work in my case, as it must change depending on the actual width of the screen.

I changed the value to the desktop design, `447px` (again, vw or % would simply not work), which makes my contact form complete for desktop as well (apart from the bg image, which most likely requires the use of media queries).

Regarding the **OPTIONAL changes**, I would kindly skip them for now and address them during the desktop implementation, due to time constraints.